app/test_split_integer.py

```diff
@@ -2,20 +2,26 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(16, 4)) == 16),\
+        "sum of the parts should be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    goal = split_integer(6, 2)
+    assert goal[0] == goal[1],\
+        "should split into equal parts when value divisible by parts"
```

Comment:
Don't use `\`. Fix it everywhere

```suggestion
    assert goal[0] == goal[1], (
        "should split into equal parts when value divisible by parts"
    )
```